alma/views/js/alma-product-insurance.js

```diff
@@ -242,9 +246,14 @@ function removeInputInsurance() {
 function addModalListenerToAddToCart() {
     if (settings.isAddToCartPopupActivated === true && almaEligibilityAnswer) {
         let addToCart = document.querySelector('.add-to-cart');
-        // If we change the quantity the DOM is reloaded then we need to remove and add the listener again
-        addToCart.removeEventListener("click", insuranceListener);
-        addToCart.addEventListener("click", insuranceListener);
+        if (!addToCart) {
```

Comment:
Isolate Add to cart selection in a method 
exemple : let addToCartButton = getAddToCartButton()